tc/sgx/trusted_worker_manager/enclave/kme/workload/kme_workload_plug-in.cpp

```diff
@@ -181,6 +192,20 @@ void KMEWorkloadProcessor::Register(
     auto search = sig_key_map.find(unique_id_bytes);
 
     if (search != sig_key_map.end()) {
+
+        // If this is a simulator mode, need not get mr_enclave
+        // from report. Rather read the one recevied from WPE.
```

Comment:
Can you mention "one received from WPE registration request"? It will be more clear.

Reply:
Reframed.